src/EnvironmentChecker.php

```diff
@@ -227,8 +244,11 @@ public function index()
     public function log($message, $level)
     {
         Injector::inst()->get(LoggerInterface::class)->log($level, $message);
+
+        return $this;
```

Comment:
```suggestion
```
Makes sense to add this for the `set*()` methods, but I don't think it makes sense for `log()`.